python/ray/train/session.py

```diff
@@ -232,6 +237,32 @@ def checkpoint(self, **kwargs):
         # checkpoint has been processed.
         self.continue_lock.acquire()
 
+    @property
+    def accelerator(self) -> Optional[Accelerator]:
```

Comment:
can you add tests for these methods in `test_session.py`?

Reply:
Fixed